metadata-transformer/src/main/java/no/unit/nva/metadata/Citation.java

```diff
@@ -0,0 +1,37 @@
+package no.unit.nva.metadata;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum Citation {
+    AUTHOR("author", DcTerms.CONTRIBUTOR),
```

Comment:
The enum defines the properties we recognise from CITATION and maps these to actual ontology values.

Reply:
An alternative approach might be to map directly to unit-namespace and avoid the SPARQL query entirely (c.f. comment below), but this incurs a slight issue when we work with other embedded data (schema.org, RDFa, Microdata, Microformats and what-have-you)